termux.md

```diff
@@ -2,34 +2,126 @@
+## Step 1a: If you don't have Termux installed
+
+You can download Termux from f-droid or the Google Play store. Get the f-droid version, since the Play store version does not support some features.  
 
 [Apk from f-droid](https://f-droid.org/it/packages/com.termux/)
-or
-[Official PlayStore](https://play.google.com/store/apps/details?id=com.termux)
+
+## Step 1b: If you have Termux installed
+
+To get this to work, we need to delete all pip packages and python
+
+```
+# Remove all python packages
+
+pip freeze > unins && pip uninstall -y -r unins && rm unins
+
+# Find the path of the cache for pip...
+
+pip cache dir
+
+# ... and clean it
+
+rm -rf /data/data/com.termux/files/home/.cache/pip
+
+# Delete python completely
+
+apt remove --purge python
+```
+
 
 ## Step 2: Install required packages
 
-    pkg upgrade -y
-	curl -LO https://its-pointless.github.io/setup-pointless-repo.sh
-	bash setup-pointless-repo.sh
-    pkg install android-tools python build-essential cmake libjpeg-turbo libpng libxml2 libxslt freetype git -y
-    pip install wheel
+```
+pkg upgrade -y
+pkg install build-essential clang make pkg-config
+curl -LO https://its-pointless.github.io/setup-pointless-repo.sh
+bash setup-pointless-repo.sh
+pkg install android-tools cmake libcompiler-rt pytz cython setuptools libjpeg-turbo libpng libxml2 libxslt freetype git libtiff -y
+
+# install numpy and scipy with pkg NOT with pip
+
+pkg install numpy
+
+pkg install scipy
+
+xport CFLAGS=" -Wno-deprecated-declarations -Wno-unreachable-code"
+
+export LDFLAGS=" -lm -lcompiler_rt"
+
+# install Pandas, this can take hours, depends on your device (I don't recommend doing this over ssh)
+
+pip install pandas
+```
 
 ## Step 3: Install GramAddict
 
+Termux puts all your files in an emulated folder structure, which can be annoying. You can install GramAddict onto a normal folder:
+
+```
+# give Termux permission to access your device's storage
+
+termux-setup-storage
+
+# cd into the non-emulated /sdcard/ folder
+
+cd /sdcard
+```
+
 This procedure is slow, use -vvv in pip if you want to see if everything installing alright
-	
-    git clone https://github.com/gramaddict/bot.git gramaddict
-    cd gramaddict
-    pip install -r requirements.txt
 
-You can also install GramAddict using pip, but in that case you won't have the config-example folder ready to go.
+```
+# download and install GramAddict
+
+git clone https://github.com/gramaddict/bot.git gramaddict
+
+cd gramaddict
+
+# I don't recommend doing this over ssh
+
+pip install -r requirements.txt
+
+pip install GramAddict[telegram-reports]
+```
     
-## Step 4: Run
-	
-    python -m uiautomator2 init
-    python run.py
+## Step 4: Config
+
+```
+# make accounts folder and folder for your account
+
+mkdir accounts
+
+mkdir accounts/[your-account-name]
+
+# download the config examples and put them in the correct folder
+
+wget -O - https://github.com/GramAddict/bot/archive/master.tar.gz | tar -xz --strip=2 "bot-master/config-examples"
```

Comment:
that's cool but we already have the config-example is we `git clone` and we can `gramaddict init your_username` if we're using pip

Reply:
I didn't have the config-example folder, after following the old installation instructions